packages/labs/virtualizer/src/layouts/shared/SizeGapPaddingBaseLayout.ts

```diff
@@ -76,9 +76,20 @@ type Padding = {[key in side]: number};
 export abstract class SizeGapPaddingBaseLayout<
   C extends SizeGapPaddingBaseLayoutConfig
 > extends BaseLayout<C> {
-  protected _itemSize: Size | {} = {};
-  protected _gaps: Gaps | {} = {};
-  protected _padding: Padding | {} = {};
+  protected _itemSize: Size = {
```

Comment:
@rictic what do you think of this?

i had to make it `Infinity` because `0` might trip up the equality checks later on (in that the initial equality check before this change was `undefined === someNumber`). so if we made it `0`, it could be the same as the new value initially.

alternatively you make it `undefined` (e.g. `this._gaps?.row`, `this._itemSize?.width`). but we end up with places like `get _gap` which then need to do `this._gaps?.row ?? whatAmI`

Reply:
Hm, I think this is a question for @graynorton